### DISCUSSION TITLE
Epoch card

### DISCUSSION THREAD

**Discussion by RayMathew** [Buzzpy/Dev-Encyclopedia#91] [Q&A] [answered]
Epoch card talks about machine learning, but it is a generic term that applies to time as well.

**Comment by Buzzpy** [comment, marked as answer]
Got it! Let's add a term like "ML" inside brackets..

**Reply by RayMathew** [reply]
This is complete https://github.com/Buzzpy/Dev-Encyclopedia/pull/99
